Feature/inference cicd stabilize 1.0 4

suhyun-kim415 opened the pull request:
<!-- This is an auto-generated comment: release notes by coderabbit.ai -->
## Summary by CodeRabbit

## 릴리스 노트

* **새로운 기능**
  * 추론 API 엔드포인트 추가: /api/v1/inference 접근 경로 제공
  * 추론 백엔드 서비스 및 배포 구성 추가로 모델 추론용 엔드포인트가 클러스터에서 서비스됨

* **기타**
  * 추론 서비스 포트 및 자원(메모리/CPU) 설정 추가 및 스케줄링 제약(노드선택자/톨러레이션/어피니티) 옵션 제공
  * 프론트엔드 및 추론 이미지 태그 업데이트

<sub>✏️ Tip: You can customize this high-level summary in your review settings.</sub>
<!-- end of auto-generated comment: release notes by coderabbit.ai -->

suhyun-kim415 committed:
feat: astrago-inference CI/CD 설정 추가

- xiilab/values.yaml에 inference 컴포넌트 추가
- helmfile/values/astrago.yaml.gotmpl에 inference 섹션 추가 (port 8000, env 설정)
- helmfile/charts/astrago/templates/inference/ 디렉토리 생성
  - service.yaml: astrago-backend-inference 서비스 생성
  - deployment.yaml: inference 배포 템플릿 생성
- helmfile/charts/astrago/templates/ingress.yaml에 inference API 엔드포인트 추가
  - /api/models, /api/servings, /api/archived-servings
- helmfile/charts/astrago/values.yaml에 inference 기본값 추가

suhyun-kim415 committed:
refactor: inference 포트 설정을 단일 소스로 통일

- helmfile/values/astrago.yaml.gotmpl에서 inference.service.port를 차트 기본값으로 직접 참조하도록 변경
- helmfile/charts/astrago/templates/ingress.yaml에서 하드코딩된 포트 8000을 변수로 변경
- 이제 helmfile/charts/astrago/values.yaml의 inference.service.port만 변경하면 모든 곳에 반영됨

lannex committed:
chore: update .coderabbit.yaml configuration

- Changed language format to string
- Updated review settings and added new tools for code analysis
- Enabled web search in knowledge base and adjusted learning scope

suhyun-kim415 committed:
chore: update .coderabbit.yaml configuration

suhyun-kim415 committed:
chore: update .coderabbit.yaml configuration

suhyun-kim415 committed:
Merge branch 'stabilize/1.0' into feature/inference-cicd-stabilize-1.0_4

suhyun-kim415 committed:
refactor: inference 설정 개선

- DATABASE_URL에서 하드코딩된 default IP 주소 제거
- MODEL_PATH와 LOG_PATH를 표준 경로(/raid/models, /raid/logs)로 변경
- 템플릿 파일 끝의 불필요한 빈 줄 제거

lannex commented on helmfile/charts/astrago/templates/ingress.yaml:
```diff
@@ -60,6 +60,31 @@ spec:
+            name: astrago-backend-inference
+            port:
+              number: {{ .Values.inference.service.port }}
+
```
api를 각각 잡으면 관리가 어려운데 고정값이에요?
아니면 /api/v1/inference 이하나 deploy 이런식으로 묶는게 나아요

suhyun-kim415 replied:
vllm 주소가 나뉘어 있는것으로 알고있습니다.

lannex replied:
연훈님 변경하실테니 맞추시면됩니다.

suhyun-kim415 replied:
네 알겠습니다.

suhyun-kim415 replied:
feature/inference-cicd-stabilize-1.0_4 여기에 위에 api 통합했는데 반영이 안되네요..
다시 생성하겠습니다.

suhyun-kim415 committed:
refactor: inference ingress 경로 통합

- 개별 경로(/api/models, /api/servings, /api/archived-servings)를 하나의 경로(/api/v1/inference)로 통합
- Ingress 규칙 단순화
